trustme/__init__.py

```diff
@@ -128,6 +129,38 @@ def _identity_string_to_x509(identity):
     return x509.DNSName(alabel)
 
 
+EC_SUPPORTED = {}
+EC_SUPPORTED.update([(curve.name.upper(), curve) for curve in [
+    ec.BrainpoolP256R1,
+    ec.BrainpoolP384R1,
+    ec.BrainpoolP512R1,
+    ec.SECP192R1,
+    ec.SECP224R1,
+    ec.SECP256R1,
+    ec.SECP384R1,
+]])
+
+def _private_key(key_type, backend):
+    if isinstance(key_type, str):
+        key_type = key_type.upper()
+        m = re.match(r'^(RSA)?(\d+)$', key_type, re.IGNORECASE)
+        if m:
+            key_type = int(m.group(2))
+
+    if isinstance(key_type, int):
+        return rsa.generate_private_key(
+            public_exponent=65537,
+            key_size=key_type,
+            backend=default_backend()
+        )
+    if not isinstance(key_type, ec.EllipticCurve) and key_type in EC_SUPPORTED:
```

Comment:
Thoughts on doing a `key_type.upper() in EC_SUPPORTED` to remove case-sensitivity from curve names?

Reply:
if it's a str, this has already happened in line 145, or? Or are you talking about the keys in the dict that are updated as upper()? Confused...